### PR TITLE
added bookmark route

### PR DESCRIPTION
**Pull request by dhruvdabhi101**
Solving Issue No #26

### CODE REVIEW

File: backend/routes/bookmark.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
+
```
Comment (sanyamjain04) [inline backend/routes/bookmark.js:1]: Can you please remove the empty line in line 1.

Comment (dhruvdabhi101) [reply backend/routes/bookmark.js:1]: yeah sure

**Comment by dhruvdabhi101** [comment]
yeah..removed the line

**Review by sanyamjain04 — APPROVED** [review]
✅ LGTM
Thanks, @dhruvdabhi101 for contributing to Kubetweet. 👏🏻
__
Summary: 👌 This code looks good.

**Pull request merged**